source/jormungandr/jormungandr/instance.py

```diff
@@ -489,8 +489,22 @@ def init(self):
             logging.getLogger(__name__).debug('timeout on init for %s', self.name)
         return False
 
+    def get_street_network(self, mode, request):
+        overriden_sn_id = request.get('_street_network')
+        if overriden_sn_id:
+            def predicate(s):
+                return s.sn_system_id == overriden_sn_id
+        else:
+            def predicate(s):
+                return mode in s.modes
+        sn = next((s for s in self.street_network_services if predicate(s)), None)
+        if sn is None:
+            logging.getLogger(__name__).info('impossible to find streetnetwork module for {} ({})'.format(
```

Comment:
it crashes and I forgot to push force the correction :wink: 

now a TechnicalError is raised